data/json/items/containers.json

```diff
@@ -431,10 +431,11 @@
     "type": "GENERIC",
     "category": "container",
     "name": { "str": "small cardboard box", "str_pl": "small cardboard boxes" },
-    "description": "A small cardboard box.  No bigger than a foot in dimension.",
+    "description": "A small cardboard box.  No bigger than half a foot in dimension.",
     "weight": "151 g",
-    "volume": "1010 ml",
-    "pocket_data": [ { "pocket_type": "CONTAINER", "rigid": true, "max_contains_volume": "1 L", "max_contains_weight": "4 kg" } ],
+    "volume": "3100 ml",
+    "longest_side": "15 cm",
+    "pocket_data": [ { "pocket_type": "CONTAINER", "rigid": true, "max_contains_volume": "3000 ml", "max_contains_weight": "4 kg" } ],
```

Comment:
I'm not sure representing small box and flat box separately makes a ton of sense. 15cm cubed is a fairly unusual shape in most places we use a small box: most of the time it's things like cereal, cookies, and other packaging, where it would be more likely to have dimensions along the lines of 15cm x 3cm x 20cm or so.

At 15cm cubed, it should be able to fit items up to 30cm long on the diagonal, or at least 20cm on the diagonal flat.